### ISSUE TITLE
BCC (Bitcoin Cash) symbol in Binance exchange should be BCH

### ISSUE THREAD

**Issue by guopengliang** [ccxt-dev/ccxt#402]
In fact, most Chinese exchanges use BCC for Bitcoin Cash. It's better if we correct this in CCXT to make it less confusing.

**Comment by kroitor** [comment]
Hi, thx for opening this issue! I will correct that right away, very soon.

References: 

https://github.com/ccxt-dev/ccxt/wiki/Manual#naming-consistency
https://github.com/ccxt-dev/ccxt/issues/83
https://github.com/ccxt-dev/ccxt/issues/129

**Comment by kroitor** [comment]
This issue has been fixed in 1.9.267+. Thx again for your feedback!

**Comment by eniuz** [comment]
issue came back

**Comment by kroitor** [comment]
@eniuz can you elaborate, please?

**Comment by abagh0703** [comment]
I think he means that when you search on the Binance exchange, "BCH" shows no results but "BCC" shows a result representing bitcoin cash.

**Comment by kroitor** [comment]
@abagh0703 in that case it's explained here: https://github.com/ccxt/ccxt/wiki/Manual#symbols-and-market-ids

**Comment by cookiemonster846** [comment]
Can this be fixed? All large exchanges use the BCH symbol instead of BCC.

**Comment by kroitor** [comment]
@cookiemonster846 this has been fixed long ago... Just use the most recent version of CCXT and make sure to read this carefully: https://github.com/ccxt/ccxt/wiki/Manual#symbols-and-market-ids

**Comment by jjaone** [comment]
No, it has not been fixed. The Binance official Android app doesn't recognize BCH, but only shows BCC in every trading related view and tab.

**Comment by jjaone** [comment]
@kroitor The topic of this issue is 'bitcoin cash' symbol in Binance exchange should be BCH".. thus I commented that in Binance exhange viewed either via their website or official Android app that coin's symbol is BCC everywhere, and not BCH as it should.

Thus, it has not been fixed (in Binance).

**Comment by kroitor** [comment]
> Thus, it has not been fixed (in Binance).

Yes, in Binance it has not been fixed, but if you look into other exchanges as well, they have all sorts of inconsistencies with symbols, as described by those links above. And we have fixtures and remapping for those cases to bring them to consistency within CCXT. So we don't see it as an issue with the CCXT lib. Therefore, if some exchange shows a weird symbol in some of their software products – it's beyond our library, I'm afraid, and in that case we can't help, unfortunately, as we're not responsible for those exchanges' products. Hope this clarifies my previous answer.

**Comment by kroitor** [comment]
The conflict of BCC vs BCH has been fixed in CCXT implementation of Binance here: https://github.com/ccxt/ccxt/blob/master/js/binance.js#L260. 

So, *"this issue opened in CCXT has been fixed"* === *"it is fixed in CCXT"*. As for their website or mobile apps not showing symbols properly – the question about BCC should be forwarded to Binance, rather than CCXT.

**Comment by eniuz** [comment]
Sorry for late answer. I'm a developer I got lost in my editor for a while. it's obviously not an issue for someone who has time to think for a moment. is a bad UX for anybody who doesn't live in China if that matters to anybody. We're forced to live in global world with global economy. I don't remember why have I posted this here. So sorry if this is wrong place. In the same time I can see that many people know about the issue :) All the Best!
